ambry-network/src/main/java/com/github/ambry/network/http2/Http2ClientStreamStatsHandler.java

```diff
@@ -44,10 +44,10 @@ protected void channelRead0(ChannelHandlerContext ctx, Http2Frame frame) throws
     RequestInfo requestInfo = ctx.channel().attr(Http2NetworkClient.REQUEST_INFO).get();
     if (requestInfo != null) {
       requestInfo.responseFramesCount++;
-      long time = System.currentTimeMillis() - requestInfo.getStreamSendTime();
+      long time = System.currentTimeMillis() - requestInfo.getRequestSendTime();
```

Comment:
QQ: do we need to specially handle the case when requestInfo.getRequestSendTime() == -1?

Reply:
Hmm, I think this is called when we get response from server. Due to that, just thinking `requestSendTime` should have a valid time by now.